geniza/corpus/management/commands/escr_alto_to_annotation.py

```diff
@@ -101,11 +128,13 @@ def ingest_xml(self, xmlfile):
 
         # create annotations
         for tb in alto.printspace.textblocks:
-            # associate tag with tagrefs to get block type
-            tags = [tag.node.attrib for tag in alto.tags]
+            block_type = None
             if tb.block_type_id:
-                tag = next((t for t in tags if t["ID"] == tb.block_type_id), None)
-                block_type = tag["LABEL"]
+                # find first tag in tag list whose id matches block type id
```

Comment:
The sub xml object makes this more readable. I wondered when I saw the definition if you needed both the id and the label, but it seems that you do. I don't quite follow the tag ref and block type definition, but that's probably fine. Any idea, is that an escriptorium specific thing or standard alto?